components/primitives/ProgressBar/progress-bar.tsx

```diff
@@ -0,0 +1,27 @@
+import clsx from "clsx";
+import { FC, HTMLProps } from "react";
+
+interface ProgressBarProps {
+  value: string; // Provide the progress value as a number (0-100)
```

Comment:
Let this be `number` type... It's important as that's the data type that is possible we will be passing into the component.

Meanwhile, you can convert it to `string` at the point of usage in the `progress` value property.